code/game/machinery/firealarm.dm

```diff
@@ -120,8 +120,8 @@
 	obj_flags |= EMAGGED
 	update_appearance()
 	if(user)
-		user.visible_message("<span class='warning'>Sparks fly out of [src]!</span>",
-							"<span class='notice'>You emag [src], disabling its thermal sensors.</span>")
+		balloon_alert(user, "thermal sensors disabled")
+	visible_message("<span class='warning'>Sparks fly out of [src]!</span>")
```

Comment:
Can't we just ACTUALLY have sparks?

Reply:
I guess this is emag so it should be stealthy...meh.

Reply:
You can just have silent sparks

Reply:
Sparks create light and set things on fire but yeah it should have them or remove the message entirely

Reply:
Sparks to light the plasma in the room you're disabling fire alarms. Sounds almost like an intended combo.